src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -2106,9 +2101,9 @@ def __init__(
 
         self.size = size
         self.size_divisor = size_divisor
-        pad_val = pad_val or {"img": 0, "seg": 255}
+        pad_val = pad_val or {"img": 0, "mask": 0}
```

Comment:
Instance segmentation has been using `Pad` and it should be padded by 0.
To avoid conflict with instance segmentation, I had updated sseg recipes to 255.
Of course, I eventually changed using `RandomResizedCrop` instead, not using `Pad`, due to performance issue.